src/esp/gfx/PTexMeshDrawable.cpp

```diff
@@ -41,12 +43,18 @@ PTexMeshDrawable::PTexMeshDrawable(scene::SceneNode& node,
 
 void PTexMeshDrawable::draw(const Magnum::Matrix4& transformationMatrix,
                             Magnum::SceneGraph::Camera3D& camera) {
+  bool usingDrawableId =
+      static_cast<RenderCamera&>(camera).isRenderingForObjectPicking();
```

Comment:
Nit: it feels a little weird that this Drawable class knows about object picking and knows to use its drawable ID in that case. Consider renaming the Camera method from `isRenderingForObjectPicking` to `useDrawableIds`.

We might find another scenarios to render using drawable Ids, and we won't have to maintain that logic in this class; it can live in the camera class or elsewhere.

Reply:
isRenderingForObjectPicking --> useDrawableIds
Good. I think it is better. Will change.

> we won't have to maintain that logic in this class; 

May I understand what logic you are mentioning?
"objectId" is a uniform in the shader that must be set dynamically on the fly. and it happens in every "drawable" before being drawn. It cannot be set in the camera class. camera class just received the signal -- "using the drawable id in the following rendering" and conveys it to the actual drawables.

Reply:
>we won't have to maintain that logic in this class
I was just making another argument for my proposed name change. Like, maybe we want to flicker between drawable ID and semantic ID every other frame, for some debug purpose. Or we want to render with drawable ID for some reason other than object picking. All that new logic could live in the Camera class or elsewhere. This class just needs to know to check `RenderCamera::useDrawableIds`.